components/renku_data_services/repositories/apispec_base.py

```diff
@@ -15,13 +15,13 @@ class Config:
         # this rust crate does not support lookahead regex syntax but we need it in this component
         regex_engine = "python-re"
 
-    @field_validator("connection_id", mode="before", check_fields=False)
+    @field_validator("id", mode="before", check_fields=False)
     @classmethod
-    def serialize_connection_id(cls, connection_id: str | ULID | None) -> str | None:
+    def serialize_connection_id(cls, id: str | ULID | None) -> str | None:
         """Custom serializer that can handle ULIDs."""
-        if connection_id is None:
+        if id is None:
             return None
-        return str(connection_id)
+        return str(id)
```

Comment:
We have a more generic version of this which handles ULIDs in any field.

Reply:
I think, this was already there - can you tell me where to look for this generic version? Or not sure what you mean.